modules/KalturaSupport/resources/mw.KAdPlayer.js

```diff
@@ -1278,7 +1282,7 @@ mw.KAdPlayer.prototype = {
 			});
 			vid.src = source.getSrc();
 			vid.load();
-			vid.play();
+			vid.play().catch(doneCallback);

```

Comment:
need to check if we have a promise here (IE8/9/10...)

Reply:
done